.travis.yml

```diff
@@ -39,9 +39,12 @@ jobs:
 
 deploy:
   - provider: script
-    script: bash ci_tools/publish.sh "$fbn_pypi_username" "$fbn_pypi_password"
+    script: >-
+            bash ci_tools/bumpversion.sh &&
+            bash ci_tools/publish.sh "$fbn_pypi_username" "$fbn_pypi_password"
```

Comment:
Where is ci_tools/publish.sh? I don't see it here in this merge request?

Reply:
We added **ci_tools** directory in PR 108.

https://github.com/finbourne/lusid-python-tools/pull/108

This directory contains the continuous integration shell scripts used by the Travis YAML.